_docs/tasks/integrating-services-into-istio.md

```diff
@@ -1,36 +1,207 @@
 ---
 title: Integrating Services into the Mesh
 overview: This task shows you how to integrate your applications with the Istio service mesh.
-            
+
 order: 20
 
 layout: docs
 type: markdown
 ---
 
-This task shows how to do X in a Kubernetes cluster. You'll learn
-how to ...
-
+This task shows how to integrate applications with Istio into a
+Kubernetes cluster. You'll learn how to manually inject the Envoy side
+into deployments using [istioctl
+kube-inject]({{site.bareurl}}/docs/reference/istioctl/istioctl_kube-inject.html)
 
 ## Before you begin
-* Do this.
-* Do this too.
 
-## Doing ...
+This task assumes you have deployed Istio on Kubernetes.  If you have
+not done so, please first complete the [Installation
+Steps]({{site.bareurl}}/docs/tasks/istio-installation.html).
+
+## Injecting Envoy sidecar into a deployment
 
-1. Do this.
-1. Do this next. Possibly read this [related explanation](...).
+Example deployment and service to demonstrate this task. Save this as
+echo.yaml.
```

Comment:
> This one uses something simple and server side.

I added a separate deployment for the client instead of reusing echo. I also added a reference to the bookinfo sample under "what's next". I think using bookinfo itself would complicate this task since it has 6 deployments and 5 services.

> May be do a container list per pod?

I already have "Use `kubectl get deployment echo -o yaml` to inspect the modified
deployment and look for the following"  with additional notes of what the user should look for.

> May be have our debug app from manager be used as client and our app from manager test be used as a server?

We know there is no istio-specific logic in our debug apps but the user might not. To make it obvious that injection doesn't depend on apps being istio-aware I thought it better to use  well known non-istio specific images.